velox/exec/tests/OrderByTest.cpp

```diff
@@ -463,6 +463,12 @@ TEST_F(OrderByTest, spill) {
   EXPECT_LT(0, stats[0].operatorStats[1].spilledBytes);
   EXPECT_EQ(1, stats[0].operatorStats[1].spilledPartitions);
   EXPECT_EQ(2, stats[0].operatorStats[1].spilledFiles);
+  // Stats[0].operatorStats[1] is OrderBy operator and stats[0].operatorStats[0]
```

Comment:
Use PlanBuilder::capturePlanNodeId method to capture the plan node ID of OrderBy, then use toPlanStats to convert task stats to a map of plan stats, then remove this comment.

Reply:
Updated.